src/test_runner/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is a stub lib.rs.
+
+#![feature(test)]
+#![feature(custom_test_frameworks)]
+pub mod test_runner;
+
+extern crate test;
```

Comment:
`extern crate test` will interfere with sanitizers. Better to enable this only in test mode. e.g.,

```
#[cfg(test)]
extern crate test;
```

Not sure whether Rust supports such syntax?

Reply:
Yes, It can. But I can not enable `#![test_runner(risingwave_test_runner::test_runner::run_failpont_tests)]
` only in test mode

Reply:
You may move risingwave_test_runner to dev-dependencies of risingwave_storage?

Reply:
I try it , Thank you very much